### PR TITLE
Updated example using async/await

### PR DESCRIPTION
**Pull request by chuacw**
Updated example using async/await, which is much easier to read.

### CODE REVIEW

File: public/docs/getting_started/javascript-tests.md [hunk 1]
```diff
@@ -83,6 +83,61 @@ contract('MetaCoin', function(accounts) {
 });
 ```
 
+## Example (using async/await)
+```
+const MetaCoin = artifacts.require("MetaCoin");
+
+contract('2nd MetaCoin test', async (accounts) => {
+
+  it("should put 10000 MetaCoin in the first account", async() => {
```
Comment (gnidan) [inline public/docs/getting_started/javascript-tests.md]: Style suggestion for consistency: space after `async`

File: public/docs/getting_started/javascript-tests.md [hunk 2]
```diff
@@ -83,6 +83,61 @@ contract('MetaCoin', function(accounts) {
...
+     let instance = await MetaCoin.deployed();
+     let balance = await instance.getBalance.call(accounts[0]);
+     assert.equal(balance.valueOf(), 10000);
+  })
+  
+  it("should call a function that depends on a linked library", async() => {
```
Comment (gnidan) [inline public/docs/getting_started/javascript-tests.md]: (same)

File: public/docs/getting_started/javascript-tests.md [hunk 3]
```diff
@@ -83,6 +83,61 @@ contract('MetaCoin', function(accounts) {
...
+    let instance = await MetaCoin.deployed();
+    let meta = instance;
```
Comment (gnidan) [inline public/docs/getting_started/javascript-tests.md]: These `instance` declarations can just be replaced by `meta` altogether, no need for the extra statement.

File: public/docs/getting_started/javascript-tests.md [hunk 4]
```diff
@@ -83,6 +83,61 @@ contract('MetaCoin', function(accounts) {
...
+    let outCoinBalance = await meta.getBalance.call(accounts[0]);
+    let metaCoinBalance = outCoinBalance.toNumber();
```
Comment (gnidan) [inline public/docs/getting_started/javascript-tests.md]: Might be worth a comment here saying that it's a BigNumber, just so `toNumber()` is more clear

**Comment by bmmpxf** [comment]
Hey there @chuacw . Great stuff. I quite like having the two different examples there. I merged your PR and updated the existing example so that they match up with what is currently in the code. Thanks for thes update!

**Comment by bmmpxf** [comment]
And thanks @gnidan for the review. :)

**Pull request merged**